src/commands/ban.js

```diff
@@ -21,19 +21,33 @@ async function banHandler(interaction) {
 	const userIds = [...new Set(Array.from(users.matchAll(Discord.MessageMentions.USERS_PATTERN), match => match[1]))];
 
 	const bansListEmbed = new Discord.MessageEmbed();
-	bansListEmbed.setTitle('User Bans :thumbsdown:');
-	bansListEmbed.setColor(0xFFA500);
+	bansListEmbed.setTitle('User Bans');
+	bansListEmbed.setColor(0xa30000);
+	bansListEmbed.setThumbnail('attachment://mod-ban.png');
+
+	const image = new Discord.MessageAttachment('./src/images/mod/mod-ban.png');
```

Comment:
Mostly a nitpick, but please try to define variables like these close to where they will be used. Right now the way code like this reads, it's not clear right away where `image` is used when going top down. If going bottom up then it's not clear where `image` comes from

Reply:
Again, I can't get as close as I wish I could, however I will rename them to be more specific.

Reply:
No you're correct. I missed where this was used inside the loop. This placement is fine